code/game/machinery/cryopod.dm

```diff
@@ -284,26 +284,25 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		to_chat(user, span_notice("[src] is already occupied!"))
 		return
 
-	if(target.stat == DEAD)
-		to_chat(user, span_notice("Dead people can not be put into cryo."))
-		return
-
-	if(target.key && user != target)
```

Comment:
You've removed the ability to put catatonic mobs (that's what the `target.key` check was for) into Cryo. This was a feature requested a while back where players wanted the ability to clean up other players who physically could not return to the shift by dropping them into Cryo and freeing up the job slot.

If there's a special reason for this, please explain and include in the PR body and changelog.